CONTRIBUTING.rst

```diff
@@ -124,7 +124,7 @@ available options to help you work with the code base.
 Before submitting a PR, please remember to ``pre-commit run --all-files``.
 But if  you forget the CI process in Github will run it for you. ;-)
 
-CircUp uses the `Click <https://click.palletsprojects.com/en/7.x/>`_ module to
+Circup uses the `Click <https://click.palletsprojects.com/en/7.x/>`_ module to
```

Comment:
Remove version, since it may already be wrong or may change.
```suggestion
Circup uses the `Click <https://click.palletsprojects.com>`_ module to
```